File: Sample.ElectronicCommerce.BrokerChat/Entities/ChatMessageEntity.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using Sample.ElectronicCommerce.Core.Entities.Base;

namespace Sample.ElectronicCommerce.BrokerChat.Entities
{
    public class ChatMessageEntity : MongoBaseEntity
    {
        public ChatMessageEntity() { }
        
        [BsonElement("id_user_sender")]
        public string IdUserSender { get; set; } = null;
        
        [BsonElement("id_user_destinatary")]
        public string IdUserDestinatary { get; set; } = null;
        
        [BsonElement("message")]
        public string Message { get; set; } = null;
    }
}

File: Sample.ElectronicCommerce.Web/ClientSide/src/app/shared/util/Entities/ChatMessageEntity.ts
export class ChatMessageEntity {
    public id: string = null; 
    public code: string = null;
    public name: string = null;
    public dtCreation: Date = null;
    public dtLastUpdate: Date = null;
    public isActive: boolean = true;
    public idUserSender: string = null;    
    public idUserDestinatary: string = null;    
    public message: string = null;
}